lib/src/auto_observable.dart

```diff
@@ -1,35 +1,20 @@
-// Copyright (c) 2013, the Dart project authors.  Please see the AUTHORS file
+// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-library observe.src.observable;
+library observe.src.auth_observable;
```

Comment:
All those years working on auth at google, and my muscle memory still kicks in :-)

Done (I kept the library just for parity with other files in the lib)